Community.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Community
+permalink: /community/
+nav_order: 99
+---
+
```

Comment:
Would definitely do a call for collaboration on tags with [Help wanted ](https://github.com/osresearch/heads/labels/help%20wanted)and [bounty](https://github.com/osresearch/heads/labels/Bounty) tags.

Or maybe better in a "Developers" section?

Reply:
Added these to the Community page and the index page of the Development section.